### ISSUE TITLE
How to optimize the training model on the existing model?

### ISSUE THREAD

**Issue by quicksandznzn** [akshaypai/tfClassifier#16]
How to optimize the training model on the existing model?

**Comment by sourcedexter** [comment]
could you explain a bit more in detail? optimized in what sense?

**Comment by DevloperHS** [comment]
You can optimize the model by trying out following things:

> Reduce batch size

> Adjust learning rate with `learning rate scheduler`

>use data augmentation 

> Use grad check for gradient decent



>
